js/info.js

```diff
@@ -41,19 +41,23 @@ define([], function() {
 
     const regenerate = function() {
         ++totalSeconds;
-        document.getElementsByClassName("info")[0].innerText = getInfo();
+        // document.getElementsByClassName("info")[0].innerText = getInfo();
```

Comment:
Please don't keep these comments if they have no use anymore

Reply:
ok, will not keep comments